### PR TITLE
Add UI

### PR DESCRIPTION
**Pull request by rbd3**
### Change made:
👉 Add app.rb
👉 Add main.rb

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    else
+      @peoples.each { |person| puts "ID: #{person.id} , Name: #{person.name}, Age: #{person.age}" }
+    end
+  end
+
+  def create_student(age, has_parent_permission, name)
+    student = Student.new(age, parent_permission: has_parent_permission, name: name)
+    @peoples << student
+  end
+
```
Comment (iambenkis) [inline app.rb:39]: - [ ] Good job creating a basic UI for your app, However , I once I wanted to create Student , He could not be saved in the `students` variable 👍 ,even though no exception was thrown 😄 . Kindly ensure that I can create a student in your app 👍

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  def list_rentals(person_id)
+    person = @peoples.find { |p| p.object_id == person_id }
+    if person.nil?
+      puts 'Person not found'
+    else
+      rentals = @rentals.select { |r| r.person.object_id == person_id }
+      rentals.each { |rental| puts "#{rental.book.title} by #{rental.book.author}" }
+    end
+  end
```
Comment (iambenkis) [inline app.rb]: - [ ] Also for `Rental`, once I wanted to create a Rental based on `Book` and `Person` , It was thrown the following exception 😢 , Kindly fix that and ensure the creation of a `Rental` is enabled 👍  

Capture

**Review by iambenkis — CHANGES_REQUESTED** [review]
#### Hi @rbd3 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
You did a great job, keep up the good work 👍🏼

giphy


## Highlights

✔️  Can create Teacher
✔️ Can create Book 
✔️  Can print list of Book and Person 

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: app.rb [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+      end
+    end
+  end
+
+  def list_people
+    if @peoples.empty?
+      puts 'There are no people'
+    else
+      @peoples.each_with_index do |person, i|
+        puts "Number: #{i + 1}, Name: #{person.name}, age: #{person.age}, ID: #{person.id}"
```
Comment (BregornOriginal) [inline app.rb:30]: - [ ] Kindly fix the UI where you list people created. There's an error where `name` it's equal to the `parent_permission` and not to the name of the person created

image

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
Hi @rbd3  

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

giphy (4)

## Highlights

✅ Linters are passing
✅ Good commit history
✅ Excellent work implementing rentals 💯 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by cynthiainga — APPROVED** [review]
Hi @rbd3, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- [ ] No linters errors ✔️ 
- [ ] All requested changes are made ✔️ 
- [ ] Correct Gitflow ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**